FunctionCMakeDoxygenFilterCompile.cmake

```diff
@@ -15,7 +15,7 @@
 #! project into a doxygen input filter executable. See
 #! http://github.com/saschazelzer/CMakeDoxygenFilter/blob/master/README for more details.
 #!
-function(FunctionCMakeDoxygenFilterCompile)
+function(FunctionCMakeDoxygenFilterCompile OUT NAMESPACE)
```

Comment:
This looks not right to me, given how the argument parsing below works. What is the intention here?